sys/netbsd/sys.txt

```diff
@@ -329,6 +350,7 @@ sched_param {
 	sched_priority	int32
 }
 
+fstatat_flags = AT_SYMLINK_NOFOLLOW, AT_SYMLINK_FOLLOW
```

Comment:
Please specify:

`#define	AT_EACCESS		0x100	/* Use euig/egid for access checks */`
`#define	AT_REMOVEDIR		0x800	/* Remove directory only */`

While there, could you please document them in the stat(2) man-page?

Reply:
Sure.